README.md

```diff
@@ -1,45 +1,4 @@
-Botsi
+slurk
```

Comment:
Should slurk be written in small or capital letters?

Reply:
I think I've mostly written it with a lower case "s". Kind of looks nicer to me, as it's not a person name.